kafka-persistence/src/main/scala/com/evolutiongaming/kafka/flow/kafkapersistence/package.scala

```diff
@@ -0,0 +1,52 @@
+package com.evolutiongaming.kafka.flow
+
+import cats.Parallel
+import cats.effect.{Concurrent, Resource, Timer}
+import com.evolutiongaming.catshelper.LogOf
+import com.evolutiongaming.kafka.flow.timer.TimersOf
+import com.evolutiongaming.kafka.journal.ConsRecord
+import com.evolutiongaming.skafka.{Offset, TopicPartition}
+import com.evolutiongaming.smetrics.MeasureDuration
+
+package object kafkapersistence {
+
+  implicit class PartitionFlowOfCompanionOps(val self: PartitionFlowOf.type) extends AnyVal {
+
+    /**
+      * Creates PartitionFlowOf which on partition assignment reads respective partition of "snapshot" (usually compacted)
+      * topic and eagerly recovers all the state from it.
+      */
+    def eagerRecoveryKafkaPersistence[F[_] : Concurrent : Timer : Parallel : MeasureDuration : LogOf, S](
```

Comment:
I see that formatting style is a bit different from the one used in Kafka Flow elsewhere. Should we try to keep it consistent / use scalafix rules or that is fine?

Reply:
In absence of scalafmt/scalafix I just used IntelliJ formatter. I'd be happy to use whatever tool to unify the format style.

Reply:
Added scalafmt but only formatted the new module so far